### PR TITLE
Bug 1866019: Delete indices in batches

### PR DESCRIPTION
**Pull request by jcantrill**
This PR:
* add error checking when looking for the "write" alias
* batches deletes to avoid cases where ES doesn't like a really long url (e.g. too long frame)

This PR may not resolve the issue as it looks like there is possibly a security exception related to an expired token.  It will add some additional info to the logs to hopefully expose more for debugging

Ref: https://bugzilla.redhat.com/show_bug.cgi?id=1866019

### CODE REVIEW

File: pkg/indexmanagement/scripts.go [hunk 1]
```diff
@@ -50,8 +56,8 @@ r=json.load(sys.stdin)
 indices = [index for index in r if int(r[index]['settings']['index']['creation_date']) < $minAgeFromEpoc ]
 if "$writeIndex" in indices:
   indices.remove("$writeIndex")
-indices.sort()
-print ','.join(indices)
+for i in range(0, len(indices), 25):
+  print ','.join(indices[i:i+25]) + " "
```
Comment (ewolinetz) [inline pkg/indexmanagement/scripts.go:60]: doesn't `', '.join(` yield you the same? the trailing `" "` looks odd to me here

Comment (jcantrill) [reply pkg/indexmanagement/scripts.go:60]: probably, experimenting in a debug pod wasn't fun so did what i could.  ill update it

File: pkg/indexmanagement/scripts.go [hunk 2]
```diff
@@ -50,8 +56,8 @@ r=json.load(sys.stdin)
 indices = [index for index in r if int(r[index]['settings']['index']['creation_date']) < $minAgeFromEpoc ]
 if "$writeIndex" in indices:
   indices.remove("$writeIndex")
-indices.sort()
-print ','.join(indices)
+for i in range(0, len(indices), 25):
+  print ', '.join(indices[i:i+25])
```
Comment (lukas-vlcek) [inline pkg/indexmanagement/scripts.go:60]: I think the trailing space should be removed. Indices are expected to be joined by comma without spacing. See https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-index.html

Comment (lukas-vlcek) [reply pkg/indexmanagement/scripts.go:60]: BTW, shouldn't the join be implemented like `strings.Join(indices,",")` ? 
`import "strings" // is needed`

**Pull request merged**